### PR TITLE
DOC-1816 - RS: Update OSS Cluster API doc

### PR DESCRIPTION
**Pull request by lanceleonard**
- Initial draft
- Also includes changes from DOC-1824 (Update prerequisites)

### CODE REVIEW

**Comment by lanceleonard** [comment]
A staged preview [is available](https://docs.redis.com/staging/jira-doc-1816/rs/databases/configure/oss-cluster-api/).

**Review by rrelledge — APPROVED** [review]
Looks good to me aside from the anchor and small typo.

File: content/rs/databases/configure/oss-cluster-api.md [hunk 1]
```diff
@@ -18,100 +18,133 @@ aliases: [
...
+
+For Active-Active (CRDB) databases, [use the crdb-cli utility](#enable-for-active-active-databases-from-the-command-line).
+
+To enable the OSS Cluster API for a Redis database from the command line:
+
+```sh
+$ rladmin tune db <database name or ID> oss_cluster enabled
+```
+
+To determine the current setting for a database from the command-line, use `rladmin info db` to return the value of the `oss_cluster` setting.
```
[the remark this answers is not in the document]
Comment (lanceleonard) [reply content/rs/databases/configure/oss-cluster-api.md]: Fixed

File: content/rs/databases/configure/oss-cluster-api.md [hunk 2]
```diff
@@ -18,100 +18,133 @@ aliases: [
...
-    ```sh
-    rladmin tune db <database name or ID> oss_cluster enabled
-    ```
+You can also use the admin console to enable the setting when creating a new database.  
+
+To do so, select **Advanced options** while creating the database in order to display the **OSS Cluster API support** setting.  Place a checkmark in the setting to enable the API when the database is created.
+
+### Command line (`rladmin`)
+
+You can use the [`rladmin` utility]({{<relref "/rs/references/cli-utilities/rladmin/">}}) to enable the OSS Cluster API for Redis Software databases, including Replica Of (Active-Passive) databases.
```
[the remark this answers is not in the document]
Comment (lanceleonard) [reply content/rs/databases/configure/oss-cluster-api.md]: Fixed.

**Pull request merged**